server/app/views/templates/base/head.scala.html

```diff
@@ -0,0 +1,33 @@
+@import play.api.Play.current
+@(title: String, description: String )(implicit request: RequestHeader)
+
+
+<meta charset="UTF-8">
+<meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+<meta name="viewport" content="width=device-width, initial-scale=1">
+<link rel="shortcut icon" type="image/png" href="@routes.Assets.at("images/favicon.png")" />
+
+<title>@title</title>
+<meta name="description" content="@description">
+<meta name="author" content="47 Degrees">
+<meta property="og:image" content="@routes.Assets.at("images/poster.png")"/>
+<meta property="og:title" content="@title"/>
+<meta property="og:site_name" content="@current.configuration.getString("application.name").getOrElse("Scala Exercises")"/>
+<meta property="og:url" content="@current.configuration.getString("application.url").getOrElse("https://www.scala-exercises.org")"/>
```

Comment:
This is not the URL we want to share. If someone shares a section this URL is always going to be the main site and not the section. We need to use the ReverseController of the given action to bring the current absolute URL here.